sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientTestsLive.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class TextAnalyticsClientTestsLive : RecordedTestBase
```

Comment:
Let's call this TextAnalyticsClientLiveTests to match TextAnalyticsClientMockTests